src/i_oplmusic.c

```diff
@@ -1891,3 +1905,7 @@ void I_OPL_DevMessages(char *result, size_t result_len)
     } while (lines < 25 && i != last_perc_count);
 }
 
+static void FinishSong(void *unused)
+{
+    I_OPL_StopSong();
```

Comment:
What if the user has chosen a different music player than OPL emulation?

Reply:
My hack only works with OPL so far.